### PR TITLE
Divide the screen and add the menu

### PR DESCRIPTION
**Pull request by HTHoussam**
On this commit i divided the screen vertically as you asked and added the menu to the top following task the folders and files structure given

### CODE REVIEW

File: src/App.module.css [hunk 1]
```diff
@@ -1,39 +1,16 @@
-* {
+.split {
```
Comment (HTHoussam) [inline src/App.module.css:1]: this is the css code responsible for dividing the screen

File: src/components/MapBoxJob.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ function MapBoxJob() {
     longitude: 8.5417,
     zoom: 8,
     height: "100vh",
-    width: "100vw",
+    width: "100%",
```
Comment (HTHoussam) [inline src/components/MapBoxJob.js:12]: gave the map 100% width to fit inside the div

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+import LeftLayout from "./components/LeftLayout";
+import TopLayout from "./components/TopLayout";
+import RightLayout from "./components/RightLayout";
+import classes from "./App.module.css";
 function App() {
-  return <MapBoxJob />;
+  return (
+    <div>
+      <TopLayout></TopLayout>
+      <div className={`${classes.split} ${classes.right}`}>
```
Comment (HTHoussam) [inline src/App.js:10]: this is the div rendered on the right of the screen contains map

**Review by HTHoussam — COMMENTED** [review]
I added some comments on my code as explanation

**Review by HT-Moh — CHANGES_REQUESTED** [review]
I am not sure why do you need to add .split class 
You can use tailwindcss please check my other comment inline

File: src/App.js [hunk 4]
```diff
@@ -2,16 +2,17 @@ import * as React from "react";
...
 function App() {
   return (
     <div>
       <TopLayout></TopLayout>
-      <div className={`${classes.split} ${classes.right}`}>
-        <RightLayout></RightLayout>
-      </div>
-      <div className={`${classes.split} ${classes.left}`}>
-        <LeftLayout></LeftLayout>
+      <div class='flex fixed w-full'>
```
Comment (HTHoussam) [inline src/App.js:9]: I used tailwind to divide instead of css as requested

**Review by HT-Moh — COMMENTED** [review]
Good Job

**Pull request merged**